example/audio-element/main.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 // Create an instance
-let wavesurfer;
+var wavesurfer; //eslint-disable-line
```

Comment:
instead of adding this eslint exception everywhere, disable the no-var rule for examples only in https://github.com/katspaugh/wavesurfer.js/blob/master/.eslintrc.js#L47

Reply:
k done